apps/heft/src/pluginFramework/IHeftPlugin.ts

```diff
@@ -9,7 +11,7 @@ import { HeftSession } from './HeftSession';
  */
 export interface IHeftPlugin<TOptions = void> {
   readonly pluginName: string;
-  readonly optionsSchemaFilePath?: string;
+  readonly optionsSchema?: JsonSchema;
```

Comment:
Really, we should export a `IJsonSchema` from NCL so it doesn't matter so much if the version of NCL that your plugin is using and the version that Heft is using don't match.

We should do that with some other things too, like `Terminal`.

Reply:
That should be a separate PR.

Reply:
Agree, but this would be a breaking change in NCL that I would consider pretty out of scope for this changeset.